pact/ledger.pact

```diff
@@ -315,21 +377,25 @@
 
     (with-default-read ledger (key id account)
       { "balance" : -1.0, "guard" : guard }
-      { "balance" := balance, "guard" := retg }
+      { "balance" := old-bal, "guard" := retg }
       (enforce (= retg guard)
         "account guards do not match")
 
-      (let ((is-new
-             (if (= balance -1.0)
-                 (enforce-reserved account guard)
-               false)))
+      (let* ((is-new
+               (if (= old-bal -1.0)
+                   (enforce-reserved account guard)
+                 false))
+              (new-bal (if is-new amount (+ old-bal amount)))
+            )
 
```

Comment:
Also add `ACCOUNT_GUARD` here? New accounts are created at mint / transfer-create / offer / buy

Reply:
I didn't realize that, thanks!

Reply:
nah, I was just tired. Thanks for spotting the bug.